docs/developer/device_crd.md

```diff
@@ -170,12 +170,13 @@ spec:
 ## Device Mapper
 
  Mapper is an application that is used to connect and control devices. The following are the responsibilities of the mapper:
- 1) Scan and connect to the device.
- 2) Report the actual state of twin-attributes of device.
- 3) Map the expected state of device-twin to the actual state of device-twin.
- 4) Collect telemetry data from the device.
- 5) Convert readings from the device to a format accepted by KubeEdge.
- 6) Schedule actions on the device.
+
+ 1. Scan and connect to the device.
+ 2. Report the actual state of twin-attributes of device.
+ 3. Map the expected state of device-twin to the actual state of device-twin.
+ 4. Collect telemetry data from the device.
+ 5. Convert readings from the device to a format accepted by KubeEdge.
+ 6. Schedule actions on the device.
```

Comment:
The section was showing like the below image at https://kubeedge.io/docs/developer/device_crd#usage-of-device-crd hence I've corrected the same as well

<img width="1021" alt="Screenshot 2023-07-31 at 1 16 37 PM" src="https://github.com/kubeedge/website/assets/133650795/4e24b86c-d5c2-4126-a423-7253230248f5">